.github/workflows/build.yml

```diff
@@ -83,7 +83,7 @@ jobs:
 
     steps:
       - name: Set git to use LF
-        if: matrix.ocaml-compiler < 5.2
+        if: matrix.os == 'windows-latest' || matrix.ocaml-compiler < 5.2
```

Comment:
You're missing the point, I don't want to change LF setting in git if not required. Please keep the condition as it was before

Reply:
@smorimoto, can you open a PR to restore the previous version